### DISCUSSION TITLE
Remote TV Control senza Broadlink

### DISCUSSION THREAD

**Discussion by Osky9** [indomus/forum#145] [HUB: Home Assistant]
Ciao a tutti,
sto cercando di creare un TV remote control su HA per un Q60 Samsung. La tv è già integrata tramite Samsungtv Tizen e fin quando devo accendere e spegnere nessun problema (sto usando il classico "tap action"-"Call service" ). 
Inizio a faticare con il Source list e il resto, allora mi sono deciso a chiedere consiglio a tutti voi, vi espongo i miei dubbi: 
-Nella card predefinita non vedo tutte le "source" disponibili sulla tv, come faccio a integrarle tutte?
-Come posso inserire anche le app tipo Netflix?

-Conviene usare l'integrazione smartthings per il mio caso?

Magari faccio domande banali ma è da qualche mese che "gioco" con HA e sto cercando di farmi le ossa!
Grazie a tutti in anticipo

**Comment by snakuzzo** [comment]
Se non erro l'integrazione [Samsung Smart TV](https://www.home-assistant.io/integrations/samsungtv/) non ti permette di selezionare le app. Come hai detto anche tu, dovresti provare con l'integrazione [SmartThings](https://www.home-assistant.io/integrations/smartthings/).
Poi probabilmente in giro ci saranno anche dei custom component che ti permetteranno di fare quello che vuoi. Dai anche un'occhiata a [QUESTA DISCUSSIONE](https://community.home-assistant.io/t/improving-samsung-tv-control/151922) sul forum ufficiale di Home Assistant...vedo che, nonostante sia un topic vecchio, continuano ad arrivare contributi e feedback

**Reply by Osky9** [reply]
Grazie per le info!
In effetti dall'app SmartThings si può fare tutto quello che vorrei integrare su HA, l'unica cosa che non capisco è come andare ad integrare su lovelace le singole funzionalità, nel senso, fino ad ora uso il call.service "media_player.turn_on" e accendo la tv, invece con il componente SmartThings cosa dovrei inserire?

**Reply by snakuzzo** [reply]
Non ho device Samsung e non uso questa integrazione, quindi non posso dirtelo se non so cosa ti espone su HA.
Abilitala e vedi che cosa ti espone. Sicuramente ti esporrà una serie di servizi e di entità...ed in base a quello ti regolerai di conseguenza

**Reply by Osky9** [reply]
OK, stavo provando ad integrare SmartThings ma non mi fa procedere, mi rimanda un errore:
"Impossibile caricare il flusso di configurazione"
E non mi fa procedere...

**Comment by snakuzzo** [comment]
Mi spiace non saprei come aiutarti.
Controlla i log di HA

**Reply by Osky9** [reply]
Continuerò a lavorarci sopra! Se avrò sviluppi lo scriverò!
Grazie dell'aiuto!
